code/routers/country_page/country.js

```diff
@@ -72,7 +72,7 @@ router.post('/insert-country-info', (req, res) => {
 
 router.get('/get-country-info', (req, res) => {
 
-  name = req.body.name;
+  var name = req.body.name;
```

Comment:
So this is actually something I missed before. For GET requests we will want them to be sending data through req.query instead of req.body. This is just normal practice for GET requests. POST request are the ones that normally used the body parameter. this will be fixed when Christoph's branch is merged. until then we should wait to merge this branch because the tests will probably change.